package.json

```diff
@@ -18,6 +18,7 @@
     "mongoose": "^7.4.3",
     "next": "13.4.13",
     "postcss": "8.4.27",
+    "punycode": "^2.3.1",
```

Comment:
always ensure that your commits stay focused on the objective of the PR. Refrain from including unrelated changes in order to keep the PR clear and on track.

Reply:
revert this change